kernels/volk/volk_32f_8u_polarbutterflypuppet_32f.h

```diff
@@ -81,6 +81,8 @@ static inline void volk_32f_8u_polarbutterflypuppet_32f_generic(float* llrs,
                                                                 unsigned char* u,
                                                                 const int elements)
 {
+    (void)input;
+
```

Comment:
It's intentional that `input` is unused, so I added `(void)input` to suppress the unused parameter warnings.

Reply:
C23 provides something for this exact situation:
[`maybe_unused`](https://en.cppreference.com/w/c/language/attributes/maybe_unused)
I suggest to add a comment to the `(void)input;` lines because this will make it more verbose. Otherwise, someone may remove these lines in the future because the look superfluous.

Reply:
as to give a differing opinion: `(void)input;` is the canonical way of silencing this warning and thus of marking a variable as unused. It's as concise as it gets!

Reply:
(also, the goal here would be to enable `-Werror -Wunused-parameter` so that a removal of that line would become a build fail)